### PR TITLE
#65 Frontpage "What is Chingu" section

### LINKED ISSUE #65: Frontpage "What is Chingu" section
https://www.figma.com/file/2mKq8rdawiJO6EEVwugWYp/Chingu_Mockups?node-id=84%3A198
We need to implement the "What is Chingu" section, including an image, title, text and buttons

### PR DESCRIPTION
**Pull request by superneutrino8**
This PR adds "What is Chingu" section in Frontpage. I have used style guides from Figma design for layout. Added responsive design from Figma design. 

Resolves #65

### CODE REVIEW

File: client/components/frontpage/ChinguSection.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import Link from "next/link";
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+import { Heading2, TextBody } from "../shared/styles";
+
+const SVG = () => {
+  return (
+    <svg
```
Comment (Soreg) [inline client/components/frontpage/ChinguSection.js:9]: This SVG should probably be replaced with "home-chingu.svg" - I see that SVG is added, but I don't see any references 🙂

**Review by Soreg — CHANGES_REQUESTED** [review]
1 change request from my side regarding the SVG - Otherwise good job 👍

File: client/components/frontpage/ChinguSection.js (hunk 1 above)
Comment (superneutrino8) [reply client/components/frontpage/ChinguSection.js]: @Soreg Thank you for the feedback. I have made changes to import SVG file directly.

**Review by Soreg — APPROVED** [review]
Perfect 👍

**Review by cMikolai — APPROVED** [review]
Cool! 👍🏻

**Pull request merged**